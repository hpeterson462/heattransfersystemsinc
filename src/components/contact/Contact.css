.contact-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  align-items: center;
  margin-bottom: 5%;
}

.contact-logo {
  position: relative;
  justify-content: center;
  align-items: center;
  margin: auto;
  padding-bottom: 10px;
  width: 80px;
}

.contact-info {
  text-align: center;
  width: 300px;
  line-height: 1.5rem;
  letter-spacing: 0.1rem;
  text-transform: uppercase;
}

.contact-info-title {
  font-weight: bold;
  color: #CC0000;
}

@media only screen
and (min-device-width : 20em) 
and (max-device-width : 50em)
and (-webkit-min-device-pixel-ratio: 3)
and (orientation: portrait) {
  .contact-logo {
    padding: 10px 0 10px 0;
  }
}
