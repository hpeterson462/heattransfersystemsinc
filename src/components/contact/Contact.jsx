import React from 'react';
import logo from '../../assets/logo.png';
import './Contact.css';

export default function Contact() {

  return (
    <section>
      <h1 id="contact">Contact</h1>

      <div className="contact-container">

        <div className="contact-info">
          <p className="contact-info-title">
            Mailing Address
          </p>
          <p>332 Wright Brothers Ave.</p>
          <p>Livermore, CA 94551</p>
        </div>

        <div className="contact-info">
          <img
            className="contact-logo"
            src={logo}
            alt="Heat Transfer Systems"
          />
          <p className="contact-info-title">
            Email Address
          </p>
          <a href="mailto:info@heattransfersystems.com" className="contact-email">
            info@heattransfersystems.com
          </a>
        </div>

        <div className="contact-info">
          <p className="contact-info-title">
            Phone
          </p>
          <p>925-456-0600</p>
          <p className="contact-info-title">
            Fax
          </p>
          <p>925-456-0602</p>
        </div>

      </div>

    </section>
  )
}

