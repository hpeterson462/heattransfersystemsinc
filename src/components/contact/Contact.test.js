import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Contact from './Contact';

describe('Contact component', () => {
  afterEach(() => cleanup());

  it('renders Contact component', () => {
    const { asFragment } = render(<MemoryRouter><Contact /></MemoryRouter>);

    expect(asFragment()).toMatchSnapshot();
  });
});
