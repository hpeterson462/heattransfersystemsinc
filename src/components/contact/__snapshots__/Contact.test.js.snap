// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders Contact component 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="contact"
    >
      Contact
    </h1>
    <div
      class="contact-container"
    >
      <div
        class="contact-info"
      >
        <p
          class="contact-info-title"
        >
          Mailing Address
        </p>
        <p>
          332 Wright Brothers Ave.
        </p>
        <p>
          Livermore, CA 94551
        </p>
      </div>
      <div
        class="contact-info"
      >
        <img
          alt="Heat Transfer Systems"
          class="contact-logo"
          src="logo.png"
        />
        <p
          class="contact-info-title"
        >
          Email Address
        </p>
        <a
          class="contact-email"
          href="mailto:info@heattransfersystems.com"
        >
          info@heattransfersystems.com
        </a>
      </div>
      <div
        class="contact-info"
      >
        <p
          class="contact-info-title"
        >
          Phone
        </p>
        <p>
          925-456-0600
        </p>
        <p
          class="contact-info-title"
        >
          Fax
        </p>
        <p>
          925-456-0602
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
